web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,139 @@
 ---
-sidebar_position: 2
+# Creating a New Project
 ---
 
-# Creating a New Project
+## Choosing an approach
+
+### Use build tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use dashboard upstream:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
+
+### Installing using dashboard upstream
+- Create an empty upstream site:
+
+#### Via the Pantheon Dashboard at this link: 
+- https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65
+
+#### Or alternatively via terminus:
+
+```
+terminus site:create my-new-site "My New Site" empty --org='Decoupled Demo'
+```
+
+#### In the Pantheon Admin Dashboard:
+
+** Switch development mode to git
+
+** Click the upgrade button, and change the site plan to performance medium. 
+
+** Clone your sites repo on your local
+
+** Change into the newly cloned directory
+
+** Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+** Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+** Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
 
-## Prerequisites
+git push origin master --force
+
+```
+
+**Note:** Make sure code has synced and all active workflows are complete in the UI.
+Click on the ‘visit development site’ button to Install via the UI, selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
```

Comment:
See https://docusaurus.io/docs/next/markdown-features/admonitions for adding admonitions in the Docusaurus style
```suggestion
:::note
Make sure code has synced and all active workflows are complete in the UI.
Click on the **visit development site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
:::
```